stories/reinitialize.stories.js

```diff
@@ -0,0 +1,58 @@
+import React from "react"
+import { storiesOf } from "@storybook/react"
+
+import { useAsync } from "../packages/react-async/src"
+import DevTools from "../packages/react-async-devtools/src"
+import "./photos.css"
+
+const loadPhoto = ({ photoId }) =>
+  fetch(`https://jsonplaceholder.typicode.com/photos/${photoId}`)
+    .then(res => (res.ok ? res : Promise.reject(res)))
+    .then(res => res.json())
+
+const PhotoPlaceholder = () => (
+  <div className="photo placeholder">
+    <div className="img" />
+    <div className="title">══════</div>
+  </div>
+)
+
+const PhotoDetails = ({ data }) => (
+  <div className="photo">
+    <img className="img" src={data.thumbnailUrl} alt="" />
+    <div className="title">{data.title}</div>
+  </div>
+)
+
+const Photo = ({ photoId, ...props }) => {
```

Comment:
Here I added a story for `reinitialize`. Let me know if something needs to be changed.